src/govuk/components/components.test.js

```diff
@@ -16,8 +22,8 @@ describe('Components', () => {
     })
 
     it('renders CSS for each component', () => {
-      const sassTasks = allComponents.map((component) => {
-        const file = `${configPaths.src}/components/${component}/_${component}.scss`
+      const sassTasks = [...componentsFiles].map(([componentName, files]) => {
+        const file = files.get(`_${componentName}.scss`)?.path
```

Comment:
Notice the `?.path` [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

What if the `${componentName}.scss` file doesn't exist?

Here we continue with `undefined` so `renderSass()` has the opportunity to fail